index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hikers Point</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+    <link rel="icon" href="assets/hiker.png" type="image/gif" sizes="30x30"> 
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/crypto-js/4.1.1/crypto-js.min.js"></script>
+    <link rel="stylesheet" href="loginPage.css">
+</head>
+<body>
+    <!--Navigation bar-->
+    <header>
+        <nav class="navBar">
+            <a href="#" class="logo">
+                <img src="assets/hiker.png" alt="logo">
+                <h3>Hikers Point</h3>
+            </a>
+            <button class="loginButton" id="loginPopUpButton">Login</button>
+        </nav>
+    </header>
+    <div class="backgroundBlur"></div>
```

Comment:
naming?

Reply:
can you mention the issue regarding naming

Reply:
backgroundBlur stands for?

Reply:
baground blur effect to create a transparent div above when form popsup